src/Pipfile

```diff
@@ -13,4 +13,5 @@ gunicorn = "*"
 psycopg2-binary = "*"
 
 [dev-packages]
-django-debug-toolbar = "*"
\ No newline at end of file
+django-debug-toolbar = "*"
+nplusone = "*"
```

Comment:
What's this for?

Reply:
I believe it is a library to help detect [N+1](https://stackoverflow.com/questions/97197/what-is-the-n1-selects-problem-in-orm-object-relational-mapping) problems and other antipatterns.